### DISCUSSION TITLE
How to disable prepare_data_augmentation?

### DISCUSSION THREAD

**Discussion by johnny12150** [RUCAIBox/RecBole#765] [Q&A] [answered]
Is there a way to disable the `prepare_data_augmentation()` in `SequentialDataset`?

**Comment by hyp1231** [comment, marked as answer]
Currently, RecBole doesn't support disabling `prepare_data_augmentation`.

**Reply by johnny12150** [reply]
If I remove the `prepare_data_augmentation` by myself, will this cause any error?

**Reply by hyp1231** [reply]
Yes, the program will definitely throw errors. Currently, we don't support directly loading sequences from atomic files.
